src/Query/Builder.php

```diff
@@ -961,8 +962,8 @@ public function where($column, $operator = null, $value = null, $boolean = 'and'
             }
         }
 
-        if (func_num_args() === 1 && is_string($column)) {
-            throw new ArgumentCountError(sprintf('Too few arguments to function %s("%s"), 1 passed and at least 2 expected when the 1st is a string.', __METHOD__, $column));
+        if (func_num_args() === 1 && is_scalar($column)) {
```

Comment:
If we're expecting an array, I'd specifically check for that:

```suggestion
        if (func_num_args() === 1 && ! is_array($column)) {
```

Reply:
Fixed.